profiling/src/serializer/mod.rs

```diff
@@ -0,0 +1,6 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2023-Present Datadog, Inc.
+
+pub mod compressed_streaming_encoder;
+
+pub use compressed_streaming_encoder::*;
```

Comment:
Personally, I like either `pub mod` without using stuff, or private `mod`, with `pub use mod::*`. Any particular reason for doing both here?

Reply:
fixed